src/Utils/Api.js

```diff
@@ -14,4 +14,10 @@ export const getArticleById = (id) => {
     return newsAPI.get(`/articles/${id}`).then(({data})=> {
         return data.article
     })
-} 
\ No newline at end of file
+} 
+
+export const getArticleComments = (id) => {
+    return newsAPI.get(`/articles/${id}/comments`).then(({data}) => {
+        return data.comments
+    })
+}
```

Comment:
There are some errors appearing in the console that should be fixed (about keys etc), as well as some console logs that need to be removed before merging.